includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -141,6 +141,38 @@ public function register() {
 		add_action( 'enqueue_block_assets', [ $this, 'enqueue_editor_assets' ] );
 		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_admin_assets' ] );
 		add_action( 'edit_form_before_permalink', [ $this, 'register_generated_titles_template' ] );
+
+		add_action( 'admin_enqueue_scripts', [ $this, 'admin_enqueue_script' ] );
+	}
+
+	/**
+	 * This function should enqueue the script for the admin page.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param string $hook WordPress admin page
+	 *
+	 * @return void
+	 */
+	public function admin_enqueue_script( string $hook ): void {
+		if (
+			'tools_page_classifai' === $hook
+			&& 'language_processing' === $_GET['tab'] // phpcs:ignore
+			&& 'openai_chatgpt' === $_GET['provider'] // phpcs:ignore
```

Comment:
Same as above, `$_GET['provider']` may not always exist so we need to check for that